data/Li_2018/metadata.yml

```diff
@@ -0,0 +1,499 @@
+    in an ice-filled container for Psi leaf measurements. Psi leaf was measured using a
+    Scholander-type pressure chamber (PMS Instruments, Corvalis, OR, USA). Stomatal
+    conductance was plotted against leaf water potential and fitted with a weighted
+    polynomial regression to extract Pgs50 and Pgs90 (Table 2). All curve fittings
+    were performed using R with the fitplc package (Duursma & Choat, 2017).
+- var_in: Cbranch
+  unit_in: 1/MPa
+  trait_name: .na
+  entity_type: population
+  value_type: mean
+  basis_of_value: measurement
+  replicates: 4
+  methods: Hydraulic capacitance ^p Whole branch hydraulic capacitance (Cbranch) was
+    measured on well-watered plants using the bench dehydration method. The aboveground
+    portion of the plant was excised at predawn (when xylem tension was minimal),
+    covered by an opaque plastic bag, and transported to the lab where branch FW and
+    Psi leaf were immediately measured. Samples were allowed to slowly dehydrate under
+    lab conditions, during which time Psi stem and branch fresh mass were regularly measured.
+    Before each measurement, the branch was placed into a sealed plastic bag for roughly
+    1 hr to ensure equilibration between Psi leaf and Psi stem. Afterwards, branch samples
+    were oven dried at 70 °C until mass was constant. Branch RWC (RWCbranch, %) at
+    each measurement period was calculated and plotted against Psi stem. Cbranch was
+    estimated as the slope of the second linear portion of the water release curve,
+    expressed as DeltaRWC/DeltaPsi  (RWC MPa-1).
+- var_in: TLP
+  unit_in: MPa
+  trait_name: leaf_turgor_loss_point
+  entity_type: population
+  value_type: mean
+  basis_of_value: measurement
+  replicates: 4
+  methods: Pressure volume analysis ^p For each species, one leaf per well-watered
+    plant was collected for leaf pressure volume (PV) determination; for species with
+    reduced leaf petioles, a terminal shoot was used instead. Leaves were excised
+    under water in the early morning and transported to the lab with the petiole kept
+    under water. Samples were placed in opaque plastic bags for up to 3 hr to ensure
+    maximum rehydration. Leaf PV curves were generated following Maréchaux et al.
+    (2015). Briefly, Psi leaf was measured using the Scholander pressure chamber, and
+    fresh weight (FW) was measured with a digital balance (weighed to 0.1 mg). Then,
+    leaves were bench dried with Psi leaf, and FW measured periodically. Finally, leaves
+    were oven dried at 70 °C for at least 48 hr to obtain dry weight. Leaf PV traits
+    were estimated as described by Lenz, Wright, and Westoby (2006). The inverse of
+    leaf water potential (1/Psi leaf) was plotted against leaf relative water content
+    (RWCleaf, %) to facilitate calculation. Leaf TLP (MPa) was taken as the point
+    where the line became non-linear.
+- var_in: Ks
+  unit_in: kg/m/MPa/s
+  trait_name: sapwood_specific_conductivity
```

Comment:
this is now sapwood_specific_hydraulic_conductivity - I've changed it. It is probably a good idea to always `update from develop` before putting in pull requests to make sure we're merging in datasets that align with the current dictionary